### ISSUE TITLE
Custom Virtual Env Name Not used

### ISSUE THREAD

**Issue by jainal09** [python-poetry/poetry#8079]
- **Poetry version**: `Poetry (version 1.5.1`
- **Python version**: 
- **OS version and name**: macOS 13.4 
- **pyproject.toml**: [pyproject.toml](https://gist.github.com/jainal09/f29f84e5ae4a657be5bc6814858a9a23)

- [X] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) stable Poetry version, installed using a recommended method.
- [X] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [X] I have consulted the [FAQ](https://python-poetry.org/docs/faq/) and [blog](https://python-poetry.org/blog/) for any relevant entries or release notes.
- [X] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option) and have included the output below.

## Issue
I have the following `poetry.toml`
```
[virtualenvs]
name = "evenv"
create = true
in-project = true
```
This is the output of `❯ poetry config --list`
```
cache-dir = "/Users/jainal/Library/Caches/pypoetry"
experimental.system-git-client = false
installer.max-workers = null
installer.modern-installation = true
installer.no-binary = null
installer.parallel = true
virtualenvs.create = true
virtualenvs.in-project = true
virtualenvs.name = "evenv"
virtualenvs.options.always-copy = false
virtualenvs.options.no-pip = false
virtualenvs.options.no-setuptools = false
virtualenvs.options.system-site-packages = false
virtualenvs.path = "{cache-dir}/virtualenvs"  # /Users/jainal/Library/Caches/pypoetry/virtualenvs
virtualenvs.prefer-active-python = false
virtualenvs.prompt = "{project_name}-py{python_version}"
```
The is issue is irrespective of this config, poetry still creates the virtualenv with the name `.venv` . When it should create virtualenv `evenv`

**Comment by Secrus** [comment]
This is a bug in config listing via CLI. There is no option `virtualenv.name`. It's listed in the output of `poetry config --list` only because you put it in `poetry.toml` manually.

**Comment by jainal09** [comment]
So, is there a way to set the name of the virtualenv that poetry creates ?

**Comment by clintonroy** [comment]
No. You can manually create your virtualenv, under any name you like, and activate it, poetry will use that virtual environment then.

**Comment by jainal09** [comment]
Oh, I see. What I wanted to ask is, what about the bug in which `poetry config --list` list all the key value pairs under the 
```
[[virtualenvs]
```
for example
```
[virtualenvs]
name = "evenv"
create = true
in-project = true
xyz="a"
```
then the output of the `poetry config --list`
```
❯ poetry config  --list
virtualenvs.create = true
virtualenvs.in-project = true
virtualenvs.name = "evenv"
virtualenvs.options.always-copy = false
virtualenvs.options.no-pip = false
virtualenvs.options.no-setuptools = false
virtualenvs.options.system-site-packages = false
virtualenvs.path = "{cache-dir}/virtualenvs"  # /Users/***/Library/Caches/pypoetry/virtualenvs
virtualenvs.prefer-active-python = false
virtualenvs.prompt = "{project_name}-py{python_version}"
virtualenvs.xyz = "a"
```
